_rules/SC2-5-3-name-from-content-included-in-accessible-name.md

```diff
@@ -0,0 +1,187 @@
+---
+name: Name from content included in accessible name
+description: |
+  Interactive elements labelled through their content must have their visible label as part of their accessible name.
+
+success_criterion:
+- 2.5.3 # Label in Name
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any HTML or SVG element that:
+* has a [semantic role](#semantic-role) that is a [widget role](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [supports name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent), and 
+* has [descendants](https://www.w3.org/TR/dom41/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are [visible](#visible) [text nodes](https://www.w3.org/TR/dom/#text), and
+* has an [accessible name](#accessible-name), even if the name is the empty string ("").
+
+**Note**: [widget roles](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [support name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent) are: `button`, `checkbox`, `gridcell`, `link`, `menuitem`, `menuitemcheckbox`, `menuitemradio`, `option`, `radio`, `searchbox`, `switch`, `tab`, `treeitem`.
```

Comment:
`searchbox` seems not to be included in roles that support name from content.

Reply:
Thanks for paying attention, @audreymaniez! 

I will remove `searchbox` in the note. 
Because it's only a note and not in the normative section, the change won't require the rule to go into a second Final Call.